src/vs/workbench/services/configuration/node/configuration.ts

```diff
@@ -357,11 +353,6 @@ export class WorkspaceServiceImpl extends WorkspaceService {
 		this.baseConfigurationService = this._register(new GlobalConfigurationService(environmentService));
 	}
 
-	saveWorkspace(location: URI): TPromise<void> {
```

Comment:
@sandy081 fyi `saveWorkspace` is no longer possible from the renderer but always has to go to the main side because the ID changes and we need to reopen the workspace from the saved location